include/pacemaker.h

```diff
@@ -195,6 +195,19 @@ pcmk_query_node_name(xmlNodePtr *xml, uint32_t node_id, char **node_name,
 int pcmk_pacemakerd_status(xmlNodePtr *xml, const char *ipc_name,
                            unsigned int message_timeout_ms);
 
+/*!
+ * \brief Remove a resource
+ *
+ * \param[in] rsc_id    Resource to remove
+ * \param[in] rsc_type  Type of the resource ("primitive", "group", etc.)
+ *
+ * \return Standard Pacemaker return code
+ * \note Due to how the underlying CIB functions work, this function will
+ *       return pcmk_rc_ok even if \p rsc_type is incorrect for \p rsc_id,
+ *       or \p rsc_id doesn't exist at all.
```

Comment:
FYI, that is the desired behavior, since deleting something that doesn't exist is always successful. That's perhaps questionable if rsc_id exists with a different rsc_type, but the requested object technically doesn't exist.

Reply:
I think it's worth leaving a comment in place just to give people a heads up, but I've changed it to make it sound less like a bug.